shotover-proxy/tests/kafka_int_tests/mod.rs

```diff
@@ -485,7 +485,7 @@ async fn cluster_3_racks_multi_shotover_with_2_shotover_down(#[case] driver: Kaf
 
     // create a new connection and produce and consume messages
     let new_connection_builder = KafkaConnectionBuilder::new(driver, "localhost:9193");
-    test_cases::cluster_test_suite(&new_connection_builder).await;
+    test_cases::cluster_test_suite_with_lost_shotover_node(&new_connection_builder).await;
 
```

Comment:
Why do `cluster_3_racks_multi_shotover_with_2_shotover_down` and `cluster_3_racks_multi_shotover_with_1_shotover_missing` fail with the list groups test while `cluster_1_rack_multi_shotover_with_1_shotover_down` succeeds?

Reply:
`cluster_1_rack_multi_shotover_with_1_shotover_down` should also fail good catch, I think its just intermittent and managed to slip through CI.